operations/mimir-mixin/dashboards/tenants.libsonnet

```
@@ -530,5 +530,106 @@ local filename = 'mimir-tenants.json';
         ) +
         { legend: { show: false } },
       )
-    ),
+    )
+
+    .addRow(
+      $.row('Read Path - Query Frontend Insights')
+      .addPanel(
+        local title = 'Rate of Read Requests - Query Frontend';
+        $.panel(title) +
+        $.queryPanel(
+          'sum(rate(cortex_query_frontend_queries_total{%s, container="query-frontend", user="$user"}[$__rate_interval]))' % $.namespaceMatcher(),
+          'rate'
+        )
+      )
+      .addPanel(
+        local title = 'Rate of Read Requests - Ruler Query Frontend';
+        $.panel(title) +
+        $.queryPanel(
+          'sum(rate(cortex_query_frontend_queries_total{%s, container="ruler-query-frontend", user="$user"}[$__rate_interval]))' % $.namespaceMatcher(),
+          'rate'
+        )
+      )
+    )
+    .addRow(
+      $.row('Read Path - Query Scheduler Insights')
```

[nit] The official name is "query-scheduler". **Please replace it in the rest of the file too.**
```suggestion
      $.row('Read Path - Query-scheduler Insights')
```